### PR TITLE
Add option to assign Elastic IPs

### PR DESCRIPTION
**Pull request by fatbasstard**
This PR adds the ability to create Elastic IP's for the Loadbalancer. 

Also some sorting tweaks and removed the default image (`nginx:latest`)

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -79,17 +78,17 @@ variable "postfix" {
   description = "Postfix the role and policy names with Role and Policy"
 }
 
+variable "private_subnet_ids" {
+  type        = list(string)
+  description = "List of subnet IDs assigned to ESC cluster"
```
Comment (shoekstra) [inline variables.tf]:
```suggestion
  description = "List of subnet IDs assigned to ECS cluster"
```

File: variables.tf [hunk 2]
```diff
@@ -119,6 +118,12 @@ variable "role_policy" {
   description = "The Policy document for the role"
 }
 
+variable "secrets" {
+  type        = map
+  default     = {}
+  description = "An object representing the secret to expose to the docker container"
```
Comment (shoekstra) [inline variables.tf:124]:
```suggestion
  description = "Map containing secrets to expose to the docker container"
```

File: variables.tf [hunk 3]
```diff
@@ -62,10 +56,15 @@ variable "health_check" {
...
 variable "image" {
   type        = string
-  default     = "nginx:latest"
   description = "Docker image to run in the ECS cluster"
 }
 
+variable "loadbalancer_eip" {
+  type        = bool
+  default     = false
+  description = "Whethter to create Elastic IPs for the Loadbalancer"
```
Comment (shoekstra) [inline variables.tf:65]:
```suggestion
  description = "Whether to create Elastic IPs for the load balancer"
```

File: outputs.tf [hunk 4]
```diff
@@ -28,9 +28,9 @@ output "https_listener_arn" {
   description = "The ARN of the HTTPS listener"
 }
 
-output "tcp_listener_arn" {
-  value       = local.tcp_listener_arn
-  description = "The ARN of the TCP listener"
+output "loadbalancer_eips" {
+  value       = values(aws_eip.lb).*.public_ip
+  description = "The Elastic IPs of the loadbalancer"
```
Comment (shoekstra) [inline outputs.tf:33]:
```suggestion
  description = "The Elastic IPs of the load balancer"
```

**Review by shoekstra — CHANGES_REQUESTED** [review]

**Comment by shoekstra** [comment]
Hey @fatbasstard, can you plus add [.github/workflows/terraform.yml](https://github.com/schubergphilis/terraform-aws-mcaf-vpc/blob/master/.github/workflows/terraform.yml) to this PR while you're at it to add some basic checks.

**Comment by fatbasstard** [comment]
Will do 👍

File: github/workflows/terraform.yml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+              tf_docs_find_dir: .
+    
+      tfsec:
+        name: tfsec
+        runs-on: ubuntu-latest
+        steps:
+          - name: Check out code
+            uses: actions/checkout@master
+          - name: Terraform security scan
+            uses: triat/terraform-security-scan@v2.0.2
```
Comment (shoekstra) [inline github/workflows/terraform.yml]: Not sure what happened here but the indentation is off.

**Review by shoekstra — CHANGES_REQUESTED** [review]

**Review by shoekstra — APPROVED** [review]

**Pull request merged**